mgmtfn/k8splugin/driver.go

```diff
@@ -90,6 +92,26 @@ func netdGetNetwork(networkID string) (*mastercfg.CfgNetworkState, error) {
 	return nwCfg, nil
 }
 
+// netdGetEpg is a utility that reads the n/w oper state
+func netdGetEpg(epgName, tenantName string) (*mastercfg.EndpointGroupState, error) {
+	// Get hold of the state driver
```

Comment:
yes, there are a few netdXXX functions already here, that is why I added it here.
Let us move it to utility in another PR
https://github.com/contiv/netplugin/issues/908